pre_processing/netcdf_output_check.F90

```diff
@@ -39,7 +39,7 @@ subroutine netcdf_output_check(output_path, lwrtm_file, swrtm_file, prtm_file, &
    if (verbose) write(*,*) 'Albedo file: ', trim(alb_file)
    if (nf90_open(path=trim(adjustl(output_path))//'/'//trim(adjustl(alb_file)), &
```

Comment:
Does this mean that we're now using the default `nf90_open` from the netcdf library itself rather than `nc_open` that came from our own `./common/` netCDF routines?
I'm curious as in some place you're using `nf90_open` and in other places you're using `ncdf_open`....

Reply:
I believe I only used `nf90_open()` in `netcdf_output_check()`; if it turns up anywhere else please flag that.

In that routine, we're  opening the file just to see if it exists. Using `nc_open()` there would cause the code to exit if the file was missing, which is exactly what this routine is attempting to prevent.

Now, I really don't like this routine. It's in a stupid `for` loop and a very blunt response to a specific user's difficult network infrastructure, but it seems DWD need it.

Reply:
That's inelegant, but not your problem. Fair enough, I have no other comments on this. Thanks for the renaming!